lldb/include/lldb/Interpreter/Interfaces/ScriptedBreakpointInterface.h

```diff
@@ -0,0 +1,35 @@
+//===-- ScriptedBreakpointInterface.h -----------------------------*- C++ -*-===//
```

Comment:
Nit: new files shouldn't contain the name and filetype anymore.
```suggestion
//===----------------------------------------------------------------------===//
```